.github/workflows/test-and-build.yml

```diff
@@ -21,12 +21,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.8, 3.9]
```

Comment:
some features do not support <= 3.7 if this is a significant issue please say so